README.rst

```diff
@@ -0,0 +1,85 @@
+html-table-parser-python3
+=========================
+
+This module consists of just one small class.
+Its purpose is to parse HTML tables without help of external modules.
+Everything I use is part of python 3.
+You can install this module, or just copy the class located in *parse.py* into your own code.
+
+Getting Started
+---------------
+
+Installing
+~~~~~~~~~~
+
+Run pip install to this repo
+
+.. code:: bash
+
+ pip install git+git://github.com/schmijos/html-table-parser-python3.git
```

Comment:
The code examples need to be indented in RST (ref: [directives](http://docutils.sourceforge.net/docs/ref/rst/restructuredtext.html#directives)). Can you indent this and below code blocks to like so:

```RST
.. code:: bash

   pip install git+https://github.com/schmijos/html-table-parser-python3.git
```

Important: git:// is unencrypted/unauthenticated and should not be used.